### PR TITLE
Plugin Zoe Payments

### PR DESCRIPTION
**Pull request by williamabreu**
First version of Zoe Payments Plugin.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: ZOE_CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 Changes done in the codebase for Zoe's rules.
 
+## 9/15/2021
+
+- Zoe Payments Plugin:
+  - Added initial version
+
```
Comment (andreyrcdias) [inline ZOE_CHANGELOG.md:9]: Put this in the end of the file

**Review by andreyrcdias — CHANGES_REQUESTED** [review]
@williamabreu just a change

File: ZOE_CHANGELOG.md (hunk 1 above)
Comment (williamabreu) [reply ZOE_CHANGELOG.md:9]: Will do, thanks.

**Review by andreyrcdias — APPROVED** [review]
LGTM

**Pull request merged**